Index: src/main/java/com/yutty/cvoid2019situation/service/DetailCountService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.yutty.cvoid2019situation.service;\r\n\r\nimport com.baomidou.mybatisplus.core.metadata.IPage;\r\nimport com.baomidou.mybatisplus.extension.service.IService;\r\nimport com.yutty.cvoid2019situation.entity.Detailcount;\r\n\r\npublic interface DetailCountService extends IService<Detailcount> {\r\n\r\n<<<<<<< HEAD\r\n    IPage<Detailcount> getPage(int currentPage, int pageSize, Detailcount detailcount);\r\n=======\r\n>>>>>>> origin/b1\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/yutty/cvoid2019situation/service/DetailCountService.java b/src/main/java/com/yutty/cvoid2019situation/service/DetailCountService.java
--- a/src/main/java/com/yutty/cvoid2019situation/service/DetailCountService.java	
+++ b/src/main/java/com/yutty/cvoid2019situation/service/DetailCountService.java	
@@ -6,9 +6,8 @@
 
 public interface DetailCountService extends IService<Detailcount> {
 
-<<<<<<< HEAD
+
     IPage<Detailcount> getPage(int currentPage, int pageSize, Detailcount detailcount);
-=======
->>>>>>> origin/b1
+
 
 }
